build-tools/packages/build-tools/prettier.config.cjs

```diff
@@ -1,8 +0,0 @@
-/*!
```

Comment:
Assuming we keep the prettier checks in place (as the scripts are currently written), don't we need to keep these files?

Reply:
We could, but they're no longer needed. According to the docs, ["The configuration file will be resolved starting from the location of the file being formatted, and searching up the file tree until a config file is (or isn’t) found."](https://prettier.io/docs/en/configuration.html) Thus the one in the release group root is used. My curent position is that we should only have per-package configs in cases where we override the defaults.

Reply:
Didn't know that was supported. Nice!